scripts/cloudfront_routing.js

```
@@ -0,0 +1,54 @@
+            "/about/": "/about/index.html", // '/about/' -> '/about/index.html'
+            // Add other routes that need special handling:
+            // e.g., old paths redirected to new paths or default documents for sections.
+            // Example:
+            // "/old-page": "/new-page",      // redirect '/old-page' -> '/new-page'
+            // Internationalization (i18n) examples for language-specific homepages:
+            "/en": "/en/index.html",
+            "/en/": "/en/index.html",
+            // You can add "/fr": "/fr/index.html", "/fr/": "/fr/index.html", etc., for other languages.
+            "/swagger": "/swagger.html",    // '/swagger' -> '/swagger.html'
+            "/test": "/_next/static/media/desktop.0ec56f83.jpg"
+        };
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)